pyproject.toml

```diff
@@ -0,0 +1,55 @@
+[build-system]
+requires = ["setuptools>=42.0.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+
+[tool.black]
+line-length = 79
+target-version = ['py38', 'py39', 'py310']
+
+
+[tool.ruff]
```

Comment:
Same question as for `black`. (And I don't know `ruff`, but seems to be a linter as well?!)

Reply:
Same as above, everything in the `pyproject.toml` is copied directly from the [cookiecutter napari plugin](https://github.com/napari/cookiecutter-napari-plugin) example. So this is here because whoever set that up liked those formatting tools. We can absolutely remove this if you want.

Reply:
Thanks for explaining! It's not a big deal either way, let's leave it in for now, and I will decide what to do when we finalize the PR ;).

Reply:
Ok, I'll leave the black and ruff configuration settings in place for now.